modules/nodes-descheduler-filtering-namespace.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-filtering-namespace_{context}"]
+= Filtering pods by namespace
+
+You can configure whether or not pods are considered for eviction based on their namespace. Only the following descheduler strategies support namespace filtering:
+
+* `PodLifeTime`
+* `RemoveDuplicates`
+* `RemovePodsHavingTooManyRestarts`
+* `RemovePodsViolatingInterPodAntiAffinity`
+* `RemovePodsViolatingNodeAffinity`
+* `RemovePodsViolatingNodeTaints`
+
```

Comment:
@bergerhoffer  can we remove `RemoveDuplicates` and add `RemovePodsHavingTooManyRestarts`
RemoveDuplicates and LowNodeUtilization does not support NamespaceFiltering.

Reply:
@kasturinarra I got this list from here, and it lists `RemoveDuplicates`: https://github.com/kubernetes-sigs/descheduler#namespace-filtering

Does OpenShift's descheduler differ? CC @damemi 

Also `RemovePodsHavingTooManyRestarts` is already in the list.

Reply:
@damemi Confirmed with me that `RemoveDuplicates` was only recently added, and so not a part of 4.6. Removing!

Reply:
Thanks !!